helm/kube-prometheus/charts/exporter-kube-api/templates/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ template "name" . }}
+    component: kube-api
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  name: {{ template "fullname" . }}
+  namespace: kube-system
+spec:
+  clusterIP: None
+  ports:
+    - name: https-metrics
+      port: 443
+      protocol: TCP
+      targetPort: 443
```

Comment:
While this is a sensible default it can happen, that the apiserver is bound to a different port. For a first attempt at the helm charts this would be fine for me as it can be optimized in further PRs. @mgoodness' call here, I believe he said that he is preparing to release some charts some time soon.